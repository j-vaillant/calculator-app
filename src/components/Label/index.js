import React from 'react';
import styled from 'styled-components';

const Label = styled.span`
display: inline-block;
flex 1 0;
font-size: 2rem;
 text-align: ${props => props.align};
`;

export default ({ text, align = 'left' }) => (
  <Label align={align}>{text}</Label>
)
