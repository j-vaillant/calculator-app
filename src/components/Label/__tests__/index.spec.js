import React from 'react';
import { shallow } from 'enzyme';

import Label from '../';


describe('Label', () => {
  it('should render correctly', () => {
    const tree = shallow(<Label text="foo" />);

   expect(tree).toMatchSnapshot();
  });
});
