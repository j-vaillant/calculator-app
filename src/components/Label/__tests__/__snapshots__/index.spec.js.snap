// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should render correctly 1`] = `
<styled.span
  align="left"
>
  foo
</styled.span>
`;
